monai/apps/pathology/data/datasets.py

```diff
@@ -119,8 +119,10 @@ class SmartCachePatchWSIDataset(SmartCacheDataset):
             will take the minimum of (cache_num, data_length x cache_rate, data_length).
         num_init_workers: the number of worker threads to initialize the cache for first epoch.
             If num_init_workers is None then the number returned by os.cpu_count() is used.
+            If a value less than 1 is speficied, 1 will be used instead.
```

Comment:
So `0` is actually not work for the logic?

Thanks.

Reply:
Hi @Nic-Ma , yes.
```
        if self.num_workers is not None:
            self.num_workers = max(int(self.num_workers), 1)
```
The above code will convert it into 1, and without the convert, error will be raised when input 0 into `ThreadPool `.